docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/LbfgsLogisticRegressionWithOptions.cs

```diff
@@ -25,7 +25,7 @@ public static void Example()
             var options = new LbfgsLogisticRegressionBinaryTrainer.Options()
             {
                 MaximumNumberOfIterations = 100,
-                OptmizationTolerance = 1e-8f,
+                OptimizationTolerance = 1e-8f,
```

Comment:
So, this is desirable, but it is a breaking change, not sure what our procedure and thoughts are about that.

Reply:
I was going to see if we could Ask it into 1.0. I will ask.

Reply:
Process: Check into master: Cherry-pick into the release branch. PR into release branch will be part of ask-mode.